### DISCUSSION TITLE
Can the system handle it like this? If RTMP doesn't connect

### DISCUSSION THREAD

**Discussion by 9rachata** [datarhei/restreamer#758] [Q&A]
Can the system handle it like this? If RTMP doesn't connect or there is no input to RTMP, can we bring images to RTMP and then come out via HLS as before? Or is there a method you can recommend? Thank you.

**Comment by ioppermann** [comment]
This kind of fallback is already on the [feature request list](https://github.com/datarhei/restreamer/discussions/345)
